### PR TITLE
Added contents to Sidecar, Ingress and DNS Record

### PR DESCRIPTION
**Pull request by sangeethah**
@alena1108 can you review the changes?

### CODE REVIEW

File: content/rancher/v2.x/en/tasks/workloads/_index.md [hunk 1]
```diff
@@ -19,15 +20,107 @@ Coming Soon
...
+
+3. When this option is used , user is presented with **Deploy Sidecar** page which has all the options from the workload deployment page.
+4. Set the **SideCar Type** to **Standard Container** or **Init Container** option which will determine the order in which 
+the sidecar container get deployed with respect to the main workload container.
+5. After entering the workload options, click on **Launch**
 
 ## Adding Ingress
 
-Coming Soon
+Ingress can be added for workloads to provide load balancing, SSL termination and name-based virtual hosting.
```
Comment (alena1108) [inline content/rancher/v2.x/en/tasks/workloads/_index.md]: `name-based virtual hosting` should be `host/path based routing`

File: content/rancher/v2.x/en/tasks/workloads/_index.md [hunk 2]
```diff
@@ -19,15 +20,107 @@ Coming Soon
...
+        Users can add rules pointing to **Service** or **Workload** by choosing the **Target Backend** **+ Service** or **+ Workload**
+        
+        For each rule , User has to add **Path** ,**Target** and **Port**
+        
+        Additional rules can added by choosing the **+ Add Rule** option.
+        
+    3. **Use as the default backend**
+        Users can add rules pointing to **Service** or **Workload** by choosing the **Target Backend** **+ Service** or **+ Workload**
+        
+        For each rule , User has to add **Path** ,**Target** and **Port**
```
Comment (alena1108) [inline content/rancher/v2.x/en/tasks/workloads/_index.md]: I don't think path is required for the default rule

File: content/rancher/v2.x/en/tasks/workloads/_index.md [hunk 3]
```diff
@@ -19,15 +20,107 @@ Coming Soon
...
+        More target ips can be added using "+ Add Target IP"
+    2. **An external hostname**
+
+        User needs to specify **Target Hostname**
+    3. **Alias of another DNS record's value**
+
+        User can add one or more DNS records using **+ Add Target Record**
+    4. **One or more workloads**
+
+        User can add one or more Workloads using **+ Add Target Workload**
```
Comment (alena1108) [inline content/rancher/v2.x/en/tasks/workloads/_index.md]: nit: `one or more existing workloads as a target`

File: content/rancher/v2.x/en/tasks/workloads/_index.md (hunk 2 above)
Comment (sangeethah) [reply content/rancher/v2.x/en/tasks/workloads/_index.md]: yes, path is optional for ```Automatically generate a .xip.io hostname``` and ```Specify a hostname to use`` and is not present for ```Use as the default backend```. Will change.

File: content/rancher/v2.x/en/tasks/workloads/_index.md (hunk 3 above)
Comment (sangeethah) [reply content/rancher/v2.x/en/tasks/workloads/_index.md]: Will include existing workloads

**Comment by alena1108** [comment]
@deniseschannon could you do a second review?

**Comment by deniseschannon** [comment]
Closing in favor of https://github.com/rancher/docs/pull/409/files

**Pull request closed without merging**